lms/djangoapps/courseware/tests/test_submitting_problems.py

```diff
@@ -599,6 +600,8 @@ def test_wrong_answers(self):
         self.check_grade_percent(0.67)
         self.assertEqual(self.get_grade_summary()['grade'], 'B')
 
+    # Temporarily skipped pending TNL-5311
```

Comment:
We've resolved this offline but for anyone reading, the plan is to remove this skip once Eric's PR has merged.